index.html

```diff
@@ -4,7 +4,7 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Should I text</title>
+  <title>Should I text,not sure why this reached me now.Things will definately reach to,for whom it has to reach to.May be after everything is closed</title>

```

Comment:
No one can help anyone ,when the time is wrong or the doors are closed.

Reply:
I cannot see you in distress like this. You can ask for help from one. You will find out soon!

Reply:
Only this helped and this alone helped me, when I was not ok.
I think you are like me in few things, I get attached quickly and I start contemplating on it a lot. Its a curse and a way to cure.

Contemplating on this and learning more bout Krishna helped me a lot. I know he will help you too!

ಕೃಷ್ಣಯೆಂದರೆ ಕಷ್ಟವು ಪರಿಹಾರ ರಾಮರಾಮ
ಕೃಷ್ಣನ ದಯದಿ ಸಕಲ ಕಷ್ಟ ಬಿಟ್ಟಿತು ರಾಮರಾಮ

Reply:
Thank you : -)

Reply:
I think you are like me in few things, I get attached quickly and I start contemplating on it a lot. Its a curse and a way to cure.


Above is true ,and the wordings will help me because the person telling has a an amazing soul.